tests/Integration/BUILD.bazel

```diff
@@ -220,6 +220,7 @@ php_gapic_library(
     srcs = ["@com_google_googleapis//google/cloud/functions/v1:functions_proto_with_info"],
     grpc_service_config = "@com_google_googleapis//google/cloud/functions/v1:functions_grpc_service_config.json",
     migration_mode = "NEW_SURFACE_ONLY",
+    transport = "grpc",
```

Comment:
Moved the "grpc-only" client to be Functions instead of Redis, since now it must be a library marked as `NEW_SURFACE_ONLY`